_data/sidebars/home_sidebar.yml

```diff
@@ -74,6 +74,14 @@ entries:
       url: /widgets/
       output: web
 
+    - title: Plugins & platform-specific code
```

Comment:
Did we come up with something to replace "platform-specific"? That's too easy to misinterpret as "cupertino vs material design".

Strawman: "Plugins for SDKs and APIs"